static/app/components/replays/walker/splitCrumbs.tsx

```diff
@@ -103,9 +103,9 @@ function SummarySegment({
 }) {
   const {handleMouseEnter, handleMouseLeave} = useCrumbHandlers(startTimestampMs);
 
-  const summaryItems = crumbs.map(crumb => (
+  const summaryItems = crumbs.map((crumb, i) => (
     <BreadcrumbItem
-      key={crumb.id}
+      key={crumb.id || i}
```

Comment:
on the index page the navigation crumbs that we use here are not fully hydrated. We've just turned the list of urls into crumbs for display reasons
(see https://github.com/getsentry/sentry/blob/master/static/app/components/replays/walker/urlWalker.tsx#L65-L77)

So react was spewing "Two children with the same key, 0" errors because that 0 is hard-coded. Array index isn't 100% perfect, there are linters out there who will warn, but it's fine here.